js/style/style.js

```diff
@@ -635,9 +665,8 @@ Style.prototype = util.inherit(Evented, {
         var sourceResults = [];
         for (var id in this.sources) {
             var source = this.sources[id];
-            if (source.queryRenderedFeatures) {
-                sourceResults.push(source.queryRenderedFeatures(queryGeometry, params, zoom, bearing));
-            }
+            var results = Source._queryRenderedVectorFeatures(source, this._layers, queryGeometry, params, zoom, bearing);
```

Comment:
@jfirebaugh I _think_ it'll just return `[]`, but I was counting on getting that straightened out during the process of reinstating the many broken unit tests